lib/producer.ex

```diff
@@ -3,12 +3,12 @@ defmodule Mississippi.Producer do
   This module defines the supervision tree of Mississippi.Producer.
   """
 
+  use Supervisor
+
   alias Mississippi.Producer.EventsProducer
   alias Mississippi.Producer.Options
 
   # Automatically defines child_spec/1
-  use Supervisor
```

Comment:
Be aware that Styler is great but has the tendency to ignoring comments: so code is moved around, while the relative comment is left at the previous line location, thus breaking the relationship between the two.